modules/candidate_list/test/candidate_listTest.php

```diff
@@ -244,22 +315,21 @@ function testDataEntryAndOpenProfile()
         //click open profile button
         $btn = self::$openProfile;
         $this->safeClick(WebDriverBy::cssSelector($btn));
-
         // input PSCID and DCCID
         $dccid = "#lorisworkspace > div > div:nth-child(1) > div >".
                  " div:nth-child(2)>form>div>div:nth-child(1)>div>div>input";
         $pscid = "#lorisworkspace > div > div:nth-child(1) > div >".
                  " div:nth-child(2)>form>div>div:nth-child(2)>div>div>input";
         // to do react input value
-        $this->webDriver->executeScript(
+        $this->webDriver->executescript(
```

Comment:
I just don't understand why we are modifying `executeScript` to `executescript` in most occurrences. I don't see any advantage but I am OK to accept, I suggest revert this kind of modifications.

Reply:
@lingma "executeScript" and "executescript" are the same in test cases. All of the other test cases use the "executescript", only these two are uppercase. That is why I changed it.

Reply:
I checked, currently not all is in uppercase, I don't see why we prefer a less advantaged solution.